packages/dev/inspector/src/inspector.ts

```diff
@@ -540,6 +548,14 @@ export class Inspector {
     }
 
     public static _CreateCanvasContainer(parentControl: HTMLElement) {
+        // If the parent control element's root is not the document (such as the ShadowRoot of the Babylon Viewer),
+        // we need to copy the styles from the document to the parent control's root.
+        if (parentControl.getRootNode() !== window.document) {
+            setTimeout(() => {
+                this._CopyStyles(window.document, parentControl.getRootNode() as unknown as DocumentOrShadowRoot);
+            }, 0);
+        }

```

Comment:
@RaananW this makes it work with the Viewer (or any situation where the Canvas (or globalRoot) is under a ShadowRoot). I think when this is called, the styles have not finished loading, because without the timeout some styles are missing and Inspector does not render correctly. I saw another part in the code (for popups I think) where it did something similar.